avro-fastserde/src/test/java/com/linkedin/avro/fastserde/FastStringableTest.java

```diff
@@ -200,14 +203,15 @@ public void deserializeStringableFields(Boolean whetherUseFastDeserializer)
     File exampleFile = new File("/tmp/test");
     URI exampleURI = new URI("urn:ISSN:1522-3611");
     URL exampleURL = new URL("http://www.example.com");
+    String exampleString = "test_string";
 
-    StringableRecord record = generateRecord(exampleURL, exampleURI, exampleFile, exampleBigDecimal, exampleBigInteger);
+    StringableRecord record = generateRecord(exampleURL, exampleURI, exampleFile, exampleBigDecimal, exampleBigInteger, exampleString);
 
     // when
     StringableRecord afterDecoding = null;
     if (whetherUseFastDeserializer) {
       afterDecoding =
-          readWithFastAvro(StringableRecord.SCHEMA$, StringableRecord.SCHEMA$, specificDataAsDecoder(record), true);
+          readWithFastAvro(StringableRecord.SCHEMA$, StringableRecord.SCHEMA$, writeWithFastAvro(record, StringableRecord.SCHEMA$, true), true);
```

Comment:
It would be nice if the test change was in a separate commit, since we cannot see what changed in the generated code... We only see that there is new generated code, but not what the previous state was.

Reply:
If you look at the diff side-by-side, you should be able to figure out what changes with the test change.
github treats those generated files as moved, so the diff is clear to me.
Let us know if it doesn't work for you.

Reply:
So are you saying that the test change resulted in zero change to the generated code? I thought the test added a new field to the schema, which should change the generated code. Otherwise, how can we see how the generated code is different, if the change touches only a part which was previously not getting tested/generated?

Reply:
I got your point.
The avro schema change did result in changes in the generated code for both serializer and de-serializer, and the change to the schema is to verify the fix.
Without the latest code gen logic, the generated fast serializer couldn't compile since it would generate something like this:
```
String stringUnion0 = ((String) data.get(9));
        if (stringUnion0 == null) {
            (encoder).writeIndex(0);
            (encoder).writeNull();
        } else {
            (encoder).writeIndex(1);
            if (stringUnion0 instanceof Utf8) {
                (encoder).writeString(((Utf8) stringUnion0));
            } else {
                (encoder).writeString(stringUnion0 .toString());
            }
        }
```
If we want to  compare the difference before/after code gen change with the change of test schema, we need to checkin the invalid java file, it could be fine since it is temporary, but I am wondering whether it is mandatory for reviewing.
As long as the new part of the generated code is reasonable for the new change to the test schema, it should be fine, right?